ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/Bls12381MulPrecompiledContract.java

```diff
@@ -0,0 +1,4 @@
+package org.hyperledger.besu.ethereum.mainnet.precompiles;
+
+public class Bls12381MulPrecompiledContract {
```

Comment:
Remove these files from the commit
They should be in the precompile directory under crosschian